be/src/exec/repeat_node.h

```diff
@@ -52,12 +55,17 @@ class RepeatNode : public ExecNode {
     std::vector<std::vector<int64_t>> _grouping_list;
     // Tuple id used for output, it has new slots.
     TupleId _output_tuple_id;
-    const TupleDescriptor* _tuple_desc;
+    const TupleDescriptor* _output_tuple_desc;
 
     std::unique_ptr<RowBatch> _child_row_batch;
     bool _child_eos;
     int _repeat_id_idx;
     RuntimeState* _runtime_state;
+
+    // Exprs used to evaluate input rows
+    std::vector<Expr*> _probe_exprs;
```

Comment:
why called `_probe_exprs` ？ there is no probe ops, you should remove probe

Reply:
done